### DISCUSSION TITLE
Bunkr Scrape Failure

### DISCUSSION THREAD

**Discussion by BSchul77** [jbsparrow/CyberDropDownloader#215] [Q&A]
So im not sure exactly what is going on. I was previously running V4, and I was encountering this same issue, so i upgraded to v5 and I am still having it. No matter what bunkr link/files I try to download, I constantly get a scrape failure. I tried going through the reading material for her, and it does state that "Quite simply, almost all of them you see will be HTTP Status codes. Such as: 404 - Not Found (dead link)" but the links are not dead.
Any help would be appreciated it

**Comment by jbsparrow** [comment]
Can you open an issue and include your log?
